Analyzer/include/Config.h

```diff
@@ -314,6 +314,27 @@ class Config
             };
             registerModules(tr, std::move(modulesList));
         }
+	else if(analyzer=="AnalyzeTopPt")
+	 {
+            const std::vector<std::string> modulesList = {
+                "PrepNTupleVars",
+                "PartialUnBlinding",
+                "PrepNTupleVars",
+                "Muon",
+                "Electron",
+                "Photon",
+                "Jet",
+                "BJet",
+                "CommonVariables",
+                "MakeMVAVariables",
+                "Baseline",
+                "DeepEventShape",
+                "BTagCorrector",
+                "ScaleFactors",
+		"TopsPt"
+            };
+	registerModules(tr, std::move(modulesList));
+        }
```

Comment:
What is this analyzer? Either remove this code or add the analyzer that goes with it. For now I suggest removing it and you can make a separate PR with this analyzer.